docs/cmf_client/cmf_client.md

```diff
@@ -275,11 +275,31 @@ Optional Arguments
 ```
 ### cmf metadata push
 ```
-Usage: cmf metadata push [-h] -p [pipeline_name] -f [file_name]  -e [exec_id]
+Usage: cmf metadata push [-h] -p [pipeline_name] -f [file_name] -e [exec_id] -t [tensorboard]
 ```
 `cmf metadata push` command pushes the metadata file from the local machine to the cmf-server.
 ```
-cmf metadata push -p 'pipeline-name' -f "/path/to/mlmd-file-name"
+cmf metadata push -p 'pipeline-name' -f '/path/to/mlmd-file-name' -e 'execution_id' -t '/path/to/tensorboard-log'
+```
+Required Arguments
+```
+-p [pipeline_name], --pipeline_name [pipeline_name]     Specify Pipeline name.
+```
+
+Optional Arguments
+```
+  -h, --help                                         show this help message and exit
+  -f [file_name],   --file_name [file_name]          Specify mlmd file name.
+  -e [exec_id],     --execution [exec_id]            Specify execution id.
+  -t [tensorboard], --tensorboard [tensorboard]      Specify path to tensorboard logs for the pipeline.
+```
+### cmf metadata export
+```
+Usage: cmf metadata export [-h] -p [pipeline_name] -j [json_file_name] -f [file_name]
+```
+`cmf metadata push` export local mlmd's metadata in json format to a json file.
```

Comment:
Is this correct ?

Reply:
Hello Ann, This is not correct. Thank you for identifying the issue. I've made the necessary corrections in the latest update. Please review the changes and let me know if any further changes are required.